tests/test_models.py

```diff
@@ -37,3 +37,14 @@ def test_load_from_json(tmpdir):
         temp_json_file.write('[{"observations":[1, 2, 3]},{"observations":[4, 5, 6]}]')
     result = load_json(example_path)
     npt.assert_array_equal(result, [[1, 2, 3], [4, 5, 6]])
+
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
```

Comment:
Add more test for:

- check for cases where there is only 1 input value
- check for cases with -ve values